data/json/recipes/food/dairy_products.json

```diff
@@ -231,5 +231,24 @@
     "qualities": [ { "id": "CHURN", "level": 1 } ],
     "components": [ [ [ "milk_cream", 8 ] ], [ [ "lemon", 2 ] ] ],
     "charges": 6
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "rennet",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "book_learn": [ [ "survival_book", 3 ], [ "dairy_book", 3 ] ],
+    "time": "30 m",
+    "batch_time_factors": [ 70, 2 ],
+    "qualities": [ { "id": "CHURN", "level": 1 } ],
+    "components": [
+      [ [ "water_clean", 8 ] ],
+      [ [ "salt", 80 ], [ "seasoning_salt", 80 ] ],
+      [ [ "vinegar", 1 ] ],
+      [ [ "young_ruminant_stomach", 1 ] ]
+    ]
   }
```

Comment:
Citing 'The Art of Natural Cheesemaking' (David Asher, 2016). This is probably the book our survivor would find to instruct them on how to make cheese. The equivalent of 'Things To Do With Milk', the book that exists in-game.

Ingredients:
1 unweaned calf, kid, or lamb.
Salt [[enough to coat the stomach at a later step, a quarter pound? 0.25kg?]]

Equipment:
Well-sharpened slaughtering, butchering, and skinning knives [[Asher seems to be assuming you will be making rennet as soon as you slaughter.]]
Stainless bowls.
Cutting boards.
2 small lengths of string
Hand pump.

Time frame:
1-2 hours to slaughter and bucher. [[Probably we're skipping this step, so I'd estimate about 30 minutes of work.]]

6 months to cure the vell [[The 'vell' is the cured, salted, preserved stomach from which rennet is extracted. We're trying to do this all in one step, so I think adding a food dehydrator and a couple hours to the recipe is reasonable.]]

Recipe
[[I'll spare you most of the details. The stomach is extracted. It is not washed. It is salted as-is. It is tied off and inflated like a balloon. It is dried. Pieces are cut off, or there is a vinegar-water soaking process to extract the rennet, which is then strained. The dried pieces of stomach can also be used on their own. If we want to add that I'm happy to expound.]]

So my recommendations with what we have in game are:
Ingredients:
Clean Water (8 units) (2 liters)
Vinegar (10 units, one seems too low)
Salt (80 is fine here)
'young ruminant stomach' (1)

Tools:
Food Dehydrator (200 charges)
"Clean Surface" quality (2) [[you're doing butchering with raw animal parts]]
"Butchering" quality (11) [[at least a stone knife]]
"Containing" quality (1) [[at least a bowl]]
Short String (2)
Cotton Patch (1) [[modeling cheesecloth for final straining]

Proficiencies:
Food Preparation
Curing and Drying Food

Timeframe: Two and a half hours.

Notably this isn't a perfect approximation of the technique outlined in the book, but is close enough to get us to our 'chymosin in a suspension of water' goal of rennet. If we're being **deathly accurate** to this recipe, we'd have to allow 'six months' for curing. (180 days!?) This seems to be mostly because of Asher's commitment to purely traditional techniques more than a strict requirement of the process. Other sources suggest ['a couple of weeks'](https://homesteadersofamerica.com/how-to-make-rennet-cheesemaking/). A dehydrator seems fine. Chymosin is not a 'living' molecule, it's just a protein, and keeps just fine in powered or dried form. Let's stick to liquid rennet for now and consider powdered for a full cheese rework some other time?